pom.xml

```diff
@@ -143,6 +144,17 @@
       <version>${kotlin.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Hva er forskjellen på disse to? Er det riktig at én av dem skal i main-kode?

Reply:
hmmm... jeg har fått inn 2... det skal bare være 1... jeg skal se på det

Reply:
org.mockito.kotlin I test-scope er satt inn og de andre fjernet...